lib/apps/customer_app/features/review/ui/review.screen.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
This `ReviewScreen` was not added to the app router so I can't open it. Read this two files to figure out how to add it:
-  https://github.com/Handees/mobile/blob/dev/lib/shared/routes/routers.dart
-  https://github.com/Handees/mobile/blob/dev/lib/shared/routes/routes.dart